QuickFixers/Controllers/LoginController.cs

```diff
@@ -50,24 +50,21 @@ public ActionResult Index(HomeViewModel homeViewModelPost)
 
                 // Add parameter names and values to dictionary
                 spParameters.Add("@Email", homeViewModelPost.Email);
-                spParameters.Add("@UserPassword", homeViewModelPost.UserPassword);
+                spParameters.Add("@UserPassword", homeViewModelPost.UserPassword.ToEncriptedString());
```

Comment:
Please rename function to `ToEncryptedString`

Reply:
updated function name